account/choices.py

```diff
@@ -0,0 +1,7 @@
+__all__ = ['Type_of_currency_CHOISES']
+
+Type_of_currency_CHOISES = (
```

Comment:
According to PEP8, constants should be written in all capital letters.
In your case:
```python
TYPE_OF_CURRENCY_CHOICES = (....)
```